v3/src/components/graph/adornments/adornment-models.ts

```diff
@@ -52,27 +52,28 @@ export const MovableValueModel = Adornment
   }))
 export interface IMovableValueModel extends Instance<typeof MovableValueModel> {}
 
+export const MovableLineParams = types.model("MovableLineParams", {
+  intercept: types.number,
+  slope: types.number,
+  pivot1: types.optional(PointModel, kInfinitePoint),
+  pivot2: types.optional(PointModel, kInfinitePoint),
+})
+
 export const MovableLineModel = Adornment
   .named('MovableLineModel')
   .props({
-    type: 'line',
-    intercept: types.number,
-    slope: types.number,
-    pivot1: types.optional(PointModel, kInfinitePoint),
-    pivot2: types.optional(PointModel, kInfinitePoint)
+    type: 'Movable Line',
+    lines: types.map(MovableLineParams)
```

Comment:
Should `lines` be something more generic like `instances` so the same property name can be used by all the adornment models?